org.eclipse.wb.swing/.gitignore

```diff
@@ -1 +1,2 @@
 /bin
+/swing2swt.jar
```

Comment:
Please remove. It is no longer needed.

Reply:
Apologies, I thought I had removed it. I have done so again. Hoping it's right this time :)